fastapi_azure_auth/user.py

```diff
@@ -1,16 +1,247 @@
+        default=None,
+        description='Signals whether the token is an app-only token',
+    )
+    in_corp: Optional[str] = Field(
+        default=None,
+        description='Signals if the client is logging in from the corporate network; if they are not, the claim is not included',
+    )
+    ipaddr: Optional[str] = Field(
+        default=None,
+        description='The IP address the user authenticated from.',
+    )
+    login_hint: Optional[str] = Field(
+        default=None,
+        description='Login hint',
+    )
+    onprem_sid: Optional[str] = Field(
+        default=None,
+        description='On-premises security identifier',
+    )
+    pwd_exp: Optional[str] = Field(
+        default=None,
+        description='The datetime at which the password expires',
+    )
+    pwd_url: Optional[str] = Field(
+        default=None,
+        description='A URL that the user can visit to change their password',
+    )
+    sid: Optional[str] = Field(
+        default=None,
+        description='Session ID, used for per-session user sign out',
+    )
+    tenant_ctry: Optional[str] = Field(
+        default=None,
+        description="Resource tenant's country/region",
+    )
+    tenant_region_scope: Optional[str] = Field(
+        default=None,
+        description='Region of the resource tenant',
+    )
+    upn: Optional[str] = Field(
+        default=None,
+        description='An identifier for the user that can be used with the username_hint parameter; not a durable identifier for the user and should not be used to key data',
+    )
+    verified_primary_email: Optional[List[str]] = Field(
+        default=None,
+        description="Sourced from the user's PrimaryAuthoritativeEmail",
+    )
+    verified_secondary_email: Optional[List[str]] = Field(
+        default=None,
+        description="Sourced from the user's SecondaryAuthoritativeEmail",
+    )
+    vnet: Optional[str] = Field(
+        default=None,
+        description='VNET specifier information',
+    )
+    xms_pdl: Optional[str] = Field(
+        default=None,
+        description='Preferred data location',
+    )
+    xms_pl: Optional[str] = Field(
+        default=None,
+        description='User-preferred language',
+    )
+    xms_tpl: Optional[str] = Field(
+        default=None,
+        description='Tenant-preferred language',
+    )
+    ztdid: Optional[str] = Field(
+        default=None,
+        description='Zero-touch Deployment ID',
+    )
+
+    # V1.0 only
+    acr: Optional[Literal['0', '1']] = Field(
+        default=None,
+        description='A value of 0 for the "Authentication context class" claim indicates the end-user authentication '
+        'did not meet the requirements of ISO/IEC 29115. Only available in V1.0 tokens',
+    )
+    # V1.0 only
+    amr: Optional[List[str]] = Field(
+        default=None,
+        description='Identifies the authentication method of the subject of the token. Only available in V1.0 tokens',
+    )
```

Comment:
Last discussion I want to raise:

Should `amr: Optional[List[str]]` be `amr: List[str] = Field(default=[]`? 🤔 

As a developer, I'd rather like to do:
```
for method in amr:
    <logic>
```
rather than:

```
if claims.amr:
    for claim in claims.amr:
         <logic>
```.

This is true for groups and a few other fields too. I'm not sure if the `None` will add any value. Thoughts?

Reply:
I think I also prefer not having the Optional but I could probably be convinced either way

Reply:
I'll convert all lists to empty lists as default

Reply:
Awesome :) This looks good to me. Think you forgot to update the test, though. 😊

Reply:
Oops, fixed the tests now